addons/stock/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,363 @@
+        SET propagate_date = FALSE
+        FROM ir_config_parameter icp
+        WHERE sr.propagate_date IS NULL
+            AND icp.key = 'stock.use_propagation_minimum_delta'
+            AND icp.value != 'True'"""
+    )
+
+
+def fill_stock_inventory_start_empty(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_inventory
+        SET start_empty = TRUE
+        WHERE {} = 'partial'
+        """.format(openupgrade.get_legacy_name('filter'))
+    )
+
+
+def fill_stock_inventory_fields(env):
+    # categ_id
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_inventory_line sil
+        SET categ_id = si.{}
+        FROM stock_inventory si
+        WHERE sil.inventory_id = si.id AND sil.categ_id IS NULL
+        """.format(openupgrade.get_legacy_name('category_id'))
+    )
+    # prod_lot_id
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_inventory_line sil
+        SET prod_lot_id = si.{}
+        FROM stock_inventory si
+        WHERE sil.inventory_id = si.id AND sil.prod_lot_id IS NULL
+        """.format(openupgrade.get_legacy_name('lot_id'))
+    )
+    # package_id
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_inventory_line sil
+        SET package_id = si.{}
+        FROM stock_inventory si
+        WHERE sil.inventory_id = si.id AND sil.package_id IS NULL
+        """.format(openupgrade.get_legacy_name('package_id'))
+    )
+    # partner_id
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_inventory_line sil
+        SET partner_id = si.{}
+        FROM stock_inventory si
+        WHERE sil.inventory_id = si.id AND sil.partner_id IS NULL
+        """.format(openupgrade.get_legacy_name('partner_id'))
+    )
+
+
+def map_stock_location_usage(env):
+    openupgrade.map_values(
+        env.cr,
+        openupgrade.get_legacy_name('usage'),
+        'usage',
+        [('procurement', 'transit'),
+         ],
+        table='stock_location',
+    )
+
+
+def fill_stock_picking_type_sequence_code(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_picking_type
+        SET sequence_code = CASE WHEN code = 'incoming' THEN 'IN'
+          WHEN code = 'outgoing' THEN 'OUT'
+          WHEN code = 'internal' AND barcode like '%-PACK' THEN 'PACK'
+          WHEN code = 'internal' AND barcode like '%-PICK' THEN 'PICK'
+          WHEN code = 'internal' AND barcode like '%-INTERNAL' THEN 'INT'
+          ELSE 'TO_FILL' END
+        WHERE sequence_code IS NULL
+        """
+    )
+
+
+def convert_many2one_stock_inventory_product_and_location(env):
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env['stock.inventory'], 'stock_inventory',
+        'location_ids', openupgrade.get_legacy_name('location_id')
+    )
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env['stock.inventory'], 'stock_inventory',
+        'product_ids', openupgrade.get_legacy_name('product_id')
+    )
```

Comment:
Check that stock_inventory_preparation_filter module is installed and `category_id`, `lot_id` fields
Related to OCA/stock-logistics-warehouse#959

Reply:
done